calculator-styles.css

```diff
@@ -0,0 +1,58 @@
+* {
+  font-family: sans-serif;
```

Comment:
You forgot to include the `box-sizing: border-box;` property here. To apply it to all elements.